.gitignore

```diff
@@ -1,5 +1,7 @@
+.puli/
 build/
 vendor/
 composer.lock
 phpspec.yml
 phpunit.xml
+puli.json
```

Comment:
Did you not commit the puli.json file by intention? Just like composer.json, this file should usually be committed and distributed with the package.